.travis.yml

```
@@ -27,10 +28,10 @@ compiler:
 before_install:
   # code should be already formatted
   - make format CLANG_FORMAT=clang-format-3.7
-  - test $(git status --porcelain | wc -l) -eq 0
+  - test $(git status --porcelain | wc -l) -eq 0 || { git diff; false; }
```

I just wanted to show the actual diff.  Otherwise it just fails silently.